core/platform/auth/firebase_auth_services_test.py

```
@@ -424,49 +412,56 @@ def revoke_refresh_tokens(self, uid: str) -> None:
+            **kwargs: Any. The properties to update. If a property is omitted,
+                then it won't be changed.
 
         Returns:
-            str. The uid of the user.
+            firebase_auth.UserRecord. The final state of the account.
 
         Raises:
             UserNotFoundError. The Firebase account has not been created yet.
         """
         if uid not in self._users_by_uid:
             raise firebase_auth.UserNotFoundError('%s not found' % uid)
```

Ruff (0.15.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)